src/GLib/MutableTypes.jl

```diff
@@ -29,7 +29,7 @@ mutable{T <: Ptr, N}(x::Array{T, N}, i = 1) = mutable(x[i])
 mutable{T}(x::Ptr{T}, i = 1) = x + (i - 1) * sizeof(T)
 mutable{T}(x::T, i) = (i == 1 ? mutable(x) : error("Object only has one element"))
 
-_addrof{T}(b::T) = ccall(:jl_value_ptr, Ptr{T}, (Ptr{Any},), &b)
+_addrof{T}(b::T) = pointer_from_objref(b)
 Base.cconvert{P <: Ptr, T}(::Type{P}, b::MutableX{T}) = isbits(T) ? convert(P, _addrof(b)) : convert(P, _addrof(b.x))
```

Comment:
`isbitstype`

Should also trigger a femtobot request on the repo, to update all of the `f{T}()` syntax to `f() where {T}`

Reply:
I had installed femtobot but it does not seem to be triggered. Is that actually possible on a branch? Or should this first be merged in order to be able running femtobot?